src/wled/models.py

```diff
@@ -287,6 +287,12 @@ class Segment(BaseModel):
     from top-left corner of the matrix.
     """
 
+    cct: int = field(default=0, metadata=field_options(alias="cct"))
```

Comment:
The alias isn't needed, as it is the same as the field name.

```suggestion
    cct: int = field(default=0)
```

Reply:
Can we assume the field is always there? As in, do we need the default?

Reply:
you are right about alias, my bad. fixed. about the default, I would leave it there just in case. Moreover, the default value is specified for other mandatory fields (eg. "on") so I suggest to remove all unnecessary default values in a different PR

Reply:
> Moreover, the default value is specified for other mandatory fields (eg. "on") 

Agreed, that is a next step. I have ported the defaults from the previous model into this one. I definitely want make it more strict in follow ups; hence me looking at this one as a new addition.

If this is a always present field in firmware version 0.14.0 and up, we should remove the default.

Reply:
The cct parameter has been added in 13.0, I think we can remove the default, are you ok?

Reply:
@frenck

Reply:
moreover, I'm not familiar with mashumaro, but what is the effect of having the property initialize? for example  `start: int = 0` do we need the `= 0` ? I cannot find any example in mashumaro docs showing properties being initialized